src/engraving/rw/write/twrite.cpp

```diff
@@ -1471,6 +1471,7 @@ void TWrite::write(const GuitarBend* item, XmlWriter& xml, WriteContext& ctx)
     xml.tag("guitarBendType", static_cast<int>(item->type()));
     xml.tag("bendStartTimeFactor", item->startTimeFactor());
     xml.tag("bendEndTimeFactor", item->endTimeFactor());
+    writeProperty(item, xml, Pid::PLAY);
```

Comment:
Similarly, should this be moved to `TWrite::writeProperties(const Spanner*`?

This would avoid the situation that the "play" property is present in the UI, but not written to the file for some types of spanners.